app/views/admin/talentlms/_form.haml

```diff
@@ -1,3 +1,4 @@
 = f.input :subdomain
 = f.input :api_key, label: 'API Key'
-= f.input :create_new_accounts, label: 'Allow creation of new accounts'
\ No newline at end of file
+= f.input :create_new_accounts, label: 'Allow creation of new accounts', hint: 'This will allow accounts to be automatically generated if they do not already exists in the LMS system.'
+= f.input :allow_automatic_redirect_to_course, label: 'Allow automatic course redirects', hint: 'This will allow bypassing the training page when only one course requires training.'
```

Comment:
```suggestion
= f.input :allow_automatic_redirect_to_course, label: 'Allow automatic course redirects', hint: 'When checked, users who only need to complete one course will be immediately redirected to TalentLMS upon login to the warehouse.  When unchecked or if a user needs to complete more than one course, a summary page will direct users to complete course with links to TalentLMS.'
```

Reply:
Included in [this commit](https://github.com/greenriver/hmis-warehouse/pull/5007/commits/c4228c3594a3b784977b32b918864b8539524568)